scripts/make_preloaded_export.py

```diff
@@ -78,6 +78,10 @@ def _export_for_copy(repo, export_dir):
         # well (DefineVisitsTask won't add them back in).
         contents.saveDimensionData("exposure",
                                    butler.registry.queryDimensionRecords("exposure"))
+        contents.saveDimensionData("group",
+                                   butler.registry.queryDimensionRecords("group"))
+        contents.saveDimensionData("instrument",
+                                   butler.registry.queryDimensionRecords("instrument"))
```

Comment:
Instrument definitely doesn't need to be manually exported (almost every dataset depends on it), but please add `day-obs` as well. I don't understand why it got transferred when `group` didn't, both should be equally unused.